GW2EIEvtcParser/EIData/Buffs/Buff.cs

```diff
@@ -322,6 +322,7 @@ public bool Available(CombatData combatData)
             new Buff("Superior Sigil of Doom", SigilOfDoom, Source.Gear, BuffClassification.Gear, BuffImages.SuperiorSigilOfDoom),
             // Runes
             new Buff("Superior Rune of the Monk", SuperiorRuneOfTheMonk, Source.Gear, BuffStackType.Stacking, 10, BuffClassification.Gear, BuffImages.SuperiorRuneOfTheMonk, 93543, GW2Builds.EndOfLife),
+            new Buff("Superior Rune of the Cavalier", SuperiorRuneOfTheCavalier, Source.Gear, BuffStackType.CappedDuration, 5, BuffClassification.Gear, BuffImages.SuperiorRuneOfTheCavalier),
```

Comment:
You can remove "BuffStackType.CappedDuration, 5", that is a force override stacking type